<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>420</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>420</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>TicTacToe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnField_0">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>360</width>
      <height>360</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnField_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>360</width>
      <height>360</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnField_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>360</width>
      <height>360</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnField_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>360</width>
      <height>360</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnField_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>360</width>
      <height>360</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnField_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>360</width>
      <height>360</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnField_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>360</width>
      <height>360</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnField_7">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>360</width>
      <height>360</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnField_8">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>360</width>
      <height>360</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
